website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -205,6 +223,30 @@ listener "tcp" {
 ```
 
 
+### Configuring custom http response headers
+
+This example shows configuring custom http response headers.
```

Comment:
If we don't have a broader description somewhere else of what custom HTTP headers are and when you might want to use them, could you add one here?

Reply:
Sounds good. I will add that here